taxi/urls.py

```diff
@@ -53,12 +51,15 @@
     ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
```

Comment:
Could you please follow the style format code? The current formatting is a bit messy.